README.txt

```diff
@@ -913,6 +913,46 @@ automatically to reach a desired rate of interrupts.
 
 
 
+VMA Extended API
```

Comment:
Generally speaking it is required to document vma_poll() and its related extra APIs.
However I do not think that README.txt is the right document. There are other extra APIs in VMA (e.g. zero copy) which are not documented here. 
vma_poll extra API deserve a document of its own with detailed specifications and sample code.
The contents of this PR will be saved however I am considering closing this PR.